budgetportal/urls.py

```diff
@@ -28,6 +28,9 @@
         '/(?P<government_slug>[\w-]+)'
         '/departments'
         '/(?P<department_slug>[\w-]+).yaml$', cache_page(CACHE_SECS)(views.department)),
+    # Department Budget Totals
```

Comment:
I think this data should be part of the Department view under a key like `programmes - we can probably replace the current `programmes` key with this. We should just ensure that it's compatible.